tweaked login and a few lines in user.js

Zane-ThummBorst opened the pull request:
Login is working smoothly and displaying errors, most get routes minus metrics seeme to be working and will block out anyone not signed in.

Zane-ThummBorst committed:
tweaked login and a few lines in user.js

Zane-ThummBorst committed:
Merge branch 'main' into more-routes

WBot3000 reviewed (commented):
This looks pretty good, though is there a reason when a user isn't logged in and they try to access a user-only page, that they get redirected to / instead of /loginlogout?

WBot3000 committed:
Redirect changes

When the user isn't logged in and is attempting to see account-only pages, they are taken to the login page instead of the main page. Also removed some inaccurate comments I left previously

WBot3000 reviewed (approved):
Looking good, looking good
